book_class.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rental
+
```

Comment:
- [ ] Kindly make the attribute `rental` have setter and getter methods. `attr_accessor :rental`. 👍